data/soapvalidator/MailClient/GAL/ZCS-3390.xml

```diff
@@ -0,0 +1,221 @@
+            </t:response>
+        </t:test>
+
+        <t:test id="CreateDomain" required="true" depends="adminlogin">
+            <t:request>
+                <CreateDomainRequest xmlns="urn:zimbraAdmin">
+                    <name>${domain}</name>
+                    <a n="zimbraNotes">Domain for testing SyncGalRequest</a>
+                </CreateDomainRequest>
+            </t:request>
+            <t:response>
+                <t:select path="//admin:CreateDomainResponse/admin:domain" attr="id" set="domain.id" />
+            </t:response>
+        </t:test>
+
+        <t:test>
+            <t:request>
+                <CreateGalSyncAccountRequest xmlns="urn:zimbraAdmin" name="${galdatasource}" type="zimbra"
+                    domain="${domain}" server="${zimbraServer.name}">
+                    <account by="name">${galaccount}@${domain}</account>
+                </CreateGalSyncAccountRequest>
+            </t:request>
+            <t:response>
+                <t:select path="//admin:CreateGalSyncAccountResponse/admin:account" attr="id" set="galaccount.id" />
+            </t:response>
+        </t:test>
+
+        <t:test id="CreateAccount1" required="true" depends="adminlogin">
+            <t:request>
+                <CreateAccountRequest xmlns="urn:zimbraAdmin">
+                    <name>${account1}@${domain}</name>
+                    <password>${defaultpassword.value}</password>
+                </CreateAccountRequest>
+            </t:request>
+            <t:response>
+                <t:select path="//admin:CreateAccountResponse/admin:account" attr="id" set="account1.id" />
+                <t:select path='//admin:CreateAccountResponse/admin:account/admin:a[@n="zimbraMailHost"]' set="account1.server" />
+            </t:response>
+        </t:test>
+
+        <t:test id="CreateAccount2" required="true" depends="adminlogin">
+            <t:request>
+                <CreateAccountRequest xmlns="urn:zimbraAdmin">
+                    <name>${account2}@${domain}</name>
+                    <password>${defaultpassword.value}</password>
+                </CreateAccountRequest>
+            </t:request>
+            <t:response>
+                <t:select path="//admin:CreateAccountResponse/admin:account" attr="id" set="account2.id" />
+                <t:select path='//admin:CreateAccountResponse/admin:account/admin:a[@n="zimbraMailHost"]' set="account2.server" />
+            </t:response>
+        </t:test>
+
+        <t:test>
+            <t:request>
+                <SyncGalAccountRequest xmlns="urn:zimbraAdmin">
+                    <account id="${galaccount.id}">
+                        <datasource by="name">${galdatasource}</datasource>
+                    </account>
+                </SyncGalAccountRequest>
+            </t:request>
+            <t:response>
+                <t:select path="//admin:SyncGalAccountResponse" />
+            </t:response>
+        </t:test>
```

Comment:
It will be a good idea to add a bit of delay for SyncGalAccountRequest request since it might take some time when full suite is executed and the server is bit loaded

Reply:
Delay is not required as it is new domain, in case we see failure we will add it.  delays are hard to clean up.. Please let me know if it fails in smoke run.